integration/js/README.md

```diff
@@ -66,6 +66,30 @@ export PATH="$PATH:$KITE_HOME/scripts/mac/path/"
 c
 ```
 
+If you run into a build error here, do the following:
+
+Credits: https://github.com/webrtc/KITE/issues/181
+1. Create the `settings.xml` file in the `.m2` directory.
```

Comment:
The only thing in my `settings.xml` is my GitHub username, and I last successfully ran KITE tests a couple of weeks ago. Are you sure you need to put springsource in here?

Reply:
Oh, I see the ticket. Why is the springsource mirror blocked?

Reply:
Not sure why it is blocked. I checked into the error but did not found much reported around KITE and then found that github issue. Also, I did not have any `settings.xml` file.

This is using the internal repository in `.m2` as a mirror rather than external, so is this not an appropriate fix here? I could not found any other solution to get KITE running locally following the setup instruction steps mentioned in the `integration/js` README.